spec/models/user_spec.rb

```diff
@@ -123,6 +123,12 @@
           expect(new_user.save).to be_falsey
           expect(new_user.errors[:email].first).to eq("has already been taken")
         end
+
+        it "does not save a duplicate email with difference capitalization" do
```

Comment:
Just a tiny grammatical tweak! It should be either `with difference in capitalization` or `with different capitalization`. Then this'll be ready 🎉

Reply:
Ah! Sorry about that. Thanks for catching it.

Reply:
No problem -- typos happen to all of us.